docs/src/pages/docs/Connecting to Databases/google-bigquery.mdx

```diff
@@ -9,7 +9,19 @@ version: 1
 ## Google BigQuery
 
 The recommended connector library for BigQuery is
-[pybigquery](https://github.com/mxmzdlv/pybigquery).
+[pybigquery](https://github.com/mxmzdlv/pybigquery). 
+
+In order to use pybigquery with docker-compose it is needed to make this dependency included in requirements. You can make this by adding file ```docker/requirements-local.txt``` with lines:
+```flask-cors
+vertica-python
+fbprophet
+sqlalchemy-vertica-python
+redis
+pybigquery
+pandas-gbq
+pydruid```
+
+With this modification all required dependencies will be installed with docker-compose up command.
```

Comment:
As this is a generic requirement for adding dependencies to `docker-compose`, I think this should not be under the BigQuery section (some of these deps are not specific to BQ).

Reply:
Just seeing this now, I agree with @villebro that this should probably live in this section instead:

https://superset.apache.org/docs/databases/installing-database-drivers

@adam-stasiak let me know if you need any help or want to discuss this further